store/store_test.go

```diff
@@ -64,6 +64,9 @@ type configTestSuite struct{}
 
 var _ = Suite(&configTestSuite{})
 
+// this is what snap.E("0") looks like when decoded into an interface{} (the /^i/ is for "interface")
+var iZeroEpoch = map[string]interface{}{"read": []interface{}{0.}, "write": []interface{}{0.}}
```

Comment:
seems some tests that used an epoch in context or action more interesting than "0" are missing?

Reply:
1. change amend test to use a local revision (d'oh)
2. make anything that would serialise back to `"0"` be `Unset`
3. add a non-0 store test
coming up soon